Example/Tests/Connection/MHVSodaConnectionTests.m

```diff
@@ -1098,7 +1100,7 @@
                 
                 it(@"should forward the error to the caller", ^
                    {
-                       [[expectFutureValue(expectedError) shouldEventually] beNonNil];
+                       [[expectFutureValue(expectedError) shouldEventuallyBeforeTimingOutAfter(3)] beNonNil];
```

Comment:
Not sure, I was having consistent errors with some of the tests, but seem to be OK now so removed